build/rpm.spec.tmpl

```diff
@@ -149,24 +149,24 @@ if test X"$RPM_INSTALL_PREFIX0" = X"" ; then
 fi
 
 cd @@PREFIX@@/service
-SERVICE_CMD=`./sync_gateway_service_install.sh --servicecmd`
+SERVICE_CMD=`./@@PRODUCT_EXEC@@_service_install.sh --servicecmd`
 
 cat <<EOF
 
-You have successfully installed @@PRODUCT_BASE_CAP@@ Sync Gateway.
+You have successfully installed @@PRODUCT_BASE_CAP@@ @@PRODUCT_EXEC@@.
 
-You can control the Couchbase Sync Gateway service by using the following command:
+You can control the Couchbase @@PRODUCT_EXEC@@ service by using the following command:
 
   ${SERVICE_CMD}
 
-That's it! Sync Gateway is now running on port 4984. We've setup a simple in-memory database
-which works great for exploring Sync Gateway's capabilities. A limited console is available
+That's it! @@PRODUCT_EXEC@@ is now running on port 4984. We've setup a simple in-memory database
+which works great for exploring @@PRODUCT_EXEC@@'s capabilities. A limited console is available
 by opening your browser to http://localhost:4985/_admin/.
 
 The command-line options are:
 
   -adminInterface=":4985": Address to bind admin interface to
-  -bucket="sync_gateway": Name of bucket
+  -bucket="@@PRODUCT_EXEC@@": Name of bucket
```

Comment:
in this case, I think it's best to leave the example bucket name as `sync_gateway`